tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/NetStandardLibraries/MySql/MySqlConnectorExerciser.cs

```diff
@@ -0,0 +1,215 @@
+﻿// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+
+using System;
+using System.Collections.Generic;
+using System.Data.Common;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using MySql.Data.MySqlClient;
```

Comment:
This test is actually testing the same MySql client as the other tests that were migrated. A project reference needs to be added for the MySQLConnector, and it needs to be used by this test file.

Reply:
And we need to make sure to do the matrix of different versions of MySQLConnector pinned to different target framework monikers like you did for MySql.Data